src/main_options.cpp

```diff
@@ -203,7 +211,8 @@ bool PrintSystemInformation()
 
     // TODO: print the OpenGL version number here
 
-    printf("SDL_ttf version (compiled): %d.%d.%d\n", SDL_TTF_MAJOR_VERSION, SDL_TTF_MINOR_VERSION, SDL_TTF_PATCHLEVEL);
+    printf("SDL_ttf version (compiled): %d.%d.%d\n",
+           SDL_TTF_MAJOR_VERSION, SDL_TTF_MINOR_VERSION, SDL_TTF_PATCHLEVEL);
```

Comment:
The same could be said here, btw, but there is no urgency in fixing this trivial thing. So nevermind.